### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ornearius** [GNS-Science/nshm-nz-opensha#18]
The project does need an acceptable license. I have requested guidance from GNS legal to see if they have views on this.
Most likely candidates for this are GPLv3, Mozilla and MIT. They main differences are in permissiveness.

**Comment by erabbott** [comment]
Thanks guys - @mcg-gns may need to be involved in this one too.

**Comment by erabbott** [comment]
And also @rachel-gns because she'll know what other issues are with legal for this project/about existing contracts and obligations

**Comment by kevinmilner** [comment]
Just an FYI, opensha will soon be moving to BSD3

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ornearius** [comment]
@kevinmilner I see that the projects are currently Apache-2.0. Is there are reason for the change?
I do not expect it will make much difference for us as they are both very permissive but it would be helpful to understand the reasoning.

**Comment by ornearius** [comment]
Talked to Jenny Woodward (GNS legal) last night. They do not have much background in this area and the main concern  was that the license is in line with our contractual obligations.
It did sound like there were not really any restrictions beyond that.

**Comment by kevinmilner** [comment]
> @kevinmilner I see that the projects are currently Apache-2.0. Is there are reason for the change?
> I do not expect it will make much difference for us as they are both very permissive but it would be helpful to understand the reasoning.

I'm not quite sure why, I just know that the USC legal folks prefer BSD for some reason and we're trying to standardize all of our licenses at SCEC.

**Comment by chrisbc** [comment]
HI Kevin - are you able to make our (your) zoom meeting today? CHeers

**Comment by chrisbc** [comment]
@chrisbc  - I agreed with @mcg-gns that I"ll produce a summary of the key features and any important differences between the main contenders. Hopefully this will help @mcg-gns to recommend something appropriate to legal/management for agreement.

**Comment by ornearius** [comment]
This was interesting https://plausible.io/blog/open-source-licenses

**Comment by chrisbc** [comment]
> This was interesting https://plausible.io/blog/open-source-licenses

Yes it is - thanks Andrew. @mcg-gns this is worth a read. I can't see an real disadvantage with using the APGL in our context and it does offer more protection than MIT while still being a FOSS license.

**Comment by mcg-gns** [comment]
yes, interesting, I will read it all again more carefully. One thing to keep in mind: we WANT people to use our code, or at least to use our science. The NSHM project is for the good of NZ and to ensure the best available science is available for decisions that impact the country. The more easy it is for people to use our models, the better the information will be for decision making in gov and private industry. So, we don't look at these as competitors like others might. This isn't a model we aim to keep to ourselves.
